### PR TITLE
8315761: Open source few swing JList and JMenuBar tests 

### PR DESCRIPTION
**Pull request by kumarabhi006**
Few closed JList and JMenubar swing tests are open sourced.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8315761](https://bugs.openjdk.org/browse/JDK-8315761): Open source few swing JList and JMenuBar tests (**Bug** - P4)

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**) ⚠️ Review applies to [cf4fe6c5](https://git.openjdk.org/jdk/pull/15621/files/cf4fe6c5fbb618dfbd9750a56daa761f3b78eaf3)
 * [Tejesh R](https://openjdk.org/census#tr) (@TejeshR13 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/15621/head:pull/15621` \
`$ git checkout pull/15621`

Update a local copy of the PR: \
`$ git checkout pull/15621` \
`$ git pull https://git.openjdk.org/jdk.git pull/15621/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 15621`

View PR using the GUI difftool: \
`$ git pr show -t 15621`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/15621.diff">https://git.openjdk.org/jdk/pull/15621.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/15621#issuecomment-1710354847)

### CODE REVIEW

File: test/jdk/javax/swing/JList/bug4832765.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import java.awt.Rectangle;
+import javax.swing.JFrame;
+import javax.swing.JList;
+import javax.swing.JScrollPane;
+import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
+
+public class bug4832765 {
+
+    static String[] data = {"One", "Two", "Three", "Four",
```
Comment (prsadhuk) [inline test/jdk/javax/swing/JList/bug4832765.java:41]: I guess data can be local var

**Review by prsadhuk — APPROVED** [review]
Minor comment...Also please ensure mach5 job is run in open too before integration

**Comment by kumarabhi006** [comment]
> Minor comment...Also please ensure mach5 job is run in open too before integration

Ran the mach5 job in open too, it was green. Link added in JBS comment.

File: test/jdk/javax/swing/JList/bug4832765.java (hunk 1 above)
Comment (kumarabhi006) [reply test/jdk/javax/swing/JList/bug4832765.java]: Updated.

**Review by TejeshR13 — APPROVED** [review]

**Pull request closed without merging**